### ISSUE TITLE
Getting null in Android when checking exif data

### ISSUE THREAD

**Issue by HashP08** [lodev09/react-native-exify#5]
i am trying to access image metadata in android by passing the image address in readAsync method but it is returning null for me.
On ios it is working fine

**Comment by lodev09** [comment]
Do you have a sample code?

**Comment by HashP08** [comment]
No, but my implementation is very basic in function call just checking the response

const result = await readAsync('https://www.geoimgr.com/images/samples/england-london-bridge.jpg')
result &&  console.log('result tags', result)
if(result){
// do operation
}

**Comment by lodev09** [comment]
Ohh... I see.. I think I missed supporting web urls and IOS does it automatically 😄 
I'll do a PR when I get a chance to fix it.. For now, you can use a File System package to save the image first and then read it

**Comment by ungaaaabungaaa** [comment]
SDK android 13+ it always returning null for geo exif data
